### PR TITLE
[UXE-5803] feat: migrate edge application functions instance creation to api v4

### PR DESCRIPTION
**Pull request by aloisio-m-bastian**
### What is the new behavior introduced by this PR?
migrate edge application functions instance creation to api v4
### Does this PR introduce breaking changes?
- [x] No
- [ ] Yes 

### Does it have a link on Figma?

<hr />

#### Make sure your pull request fits the checklist below (when applicable):

- [x] The issue title follows the format: [ISSUE_CODE] TYPE: TITLE
- [x] Commits are tagged with the right word (feat, test, refactor, etc)
- [ ] Application responsiveness was tested to different screen sizes
- [x] Code is formatted and linted
- [ ] Tags are added to the PR

#### These changes were tested on the following browsers:
- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Safari

### CODE REVIEW

**Review by lucasmendes21 — APPROVED** [review]

File: src/services/edge-application-functions-services/v4/create-function-service.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+  return parseHttpResponse(httpResponse)
+}
+
+const adapt = (payload) => {
+  return {
+    name: payload.name,
+    edge_function: payload.edgeFunctionID,
+    json_args: JSON.parse(payload.args),
+    active: true
```
Comment (pauloSF0) [inline src/services/edge-application-functions-services/v4/create-function-service.js:23]: Porque está setado como `true` ?

File: src/services/edge-application-functions-services/v4/create-function-service.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+ * @param {Object} httpResponse.body - The response body.
+ * @param {String} httpResponse.statusCode - The HTTP status code.
+ * @returns {string} The result message based on the status code.
+ * @throws {Error} If there is an error with the response.
+ */
+const parseHttpResponse = (httpResponse) => {
+  switch (httpResponse.statusCode) {
+    case 202:
+      return {
+        feedback: 'Your Function has been created'
```
Comment (pauloSF0) [inline src/services/edge-application-functions-services/v4/create-function-service.js:38]: Não seria Your `Function instance has been created` ?

File: src/services/edge-application-functions-services/v4/create-function-service.js (hunk 1 above)
Comment (aloisio-m-bastian) [reply src/services/edge-application-functions-services/v4/create-function-service.js:23]: pq é true

Comment (aloisio-m-bastian) [reply src/services/edge-application-functions-services/v4/create-function-service.js:23]: não temos campo para isso no formulário de criação e não faz sentido criar uma conta inativa

**Review by peterpaulo-azion — APPROVED** [review]

**Pull request merged**